### ISSUE TITLE
[BUG] Almalinux deployment fails to find ansible python interpreter

### ISSUE THREAD

**Issue by DSI-BenThomas** [vmware-samples/packer-examples-for-vsphere#181]
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.

### Project Version

22.03 (Default)

### VMware vSphere

7.0 Update 2

### HashiCorp Packer

1.8.0

### HashiCorp Packer Plugin for VMware vSphere

1.0.3

### Guest Operating System

Almalinux 8

### Environment Details

Cloned repo to a fresh Ubuntu 20.04 LTS deployment, ran through the pre-reqs

Alma ISO variables

```
// Removable Media Settings
iso_path           = "_isos/linux"
iso_file           = "AlmaLinux-8.5-x86_64-dvd.iso"
iso_checksum_type  = "md5"
iso_checksum_value = "f83ac836e4f4279ef903f1028263640c"

```

### Description

I've been running through each of the OS Template deployments, and so far everything has been going fine. However, I've gotten to Almalinux and while Rocky deployed without any issues, Almalinux is failing to find the ansible python interpreter at the hardcoded path of `/usr/bin/python3`

I believe it needs to point to `/usr/libexec/platform-python` instead 

### Expected Behavior

Deployment finishes successfully

### Actual Behavior

Deployment fails when gathering facts for Ansible configuration

```
   vsphere-iso.linux-almalinux: TASK [Gathering Facts] *********************************************************
    vsphere-iso.linux-almalinux: fatal: [default]: FAILED! => {"<sensitive>_facts": {}, "changed": false, "failed_modules": {"setup": {"failed": true, "module_stderr": "/bin/sh: /usr/bin/python3: No such file or directory\nShared connection to 127.0.0.1 closed.\r\n", "module_stdout": "", "msg": "The module failed to execute correctly, you probably need to set the interpreter.\nSee stdout/stderr for the exact error", "rc": 127}}, "msg": "The following modules failed to execute: setup\n"}
```

### Steps to Reproduce

1. Clone repo
2. Upload ISOs
3. Create config files
4. Start Build script
5. Select Almalinux

### Log Fragments and Files

```
Building an AlmaLinux OS 8 Template for VMware vSphere...
Initializing HashiCorp Packer and required plugins...
Starting the build....
vsphere-iso.linux-almalinux: output will be in this color.

==> vsphere-iso.linux-almalinux: Creating VM...
==> vsphere-iso.linux-almalinux: Customizing hardware...
==> vsphere-iso.linux-almalinux: Adding SATA controller...
==> vsphere-iso.linux-almalinux: Mounting ISO images...
==> vsphere-iso.linux-almalinux: Adding configuration parameters...
==> vsphere-iso.linux-almalinux: Starting HTTP server on port 8050
==> vsphere-iso.linux-almalinux: Set boot order...
==> vsphere-iso.linux-almalinux: Power on VM...
==> vsphere-iso.linux-almalinux: Waiting 2s for boot...
==> vsphere-iso.linux-almalinux: HTTP server is working at http://172.16.38.102:8050/
==> vsphere-iso.linux-almalinux: Typing boot command...
==> vsphere-iso.linux-almalinux: Waiting for IP...
==> vsphere-iso.linux-almalinux: IP address: 172.16.38.164
==> vsphere-iso.linux-almalinux: Using SSH communicator to connect: 172.16.38.164
==> vsphere-iso.linux-almalinux: Waiting for SSH to become available...
==> vsphere-iso.linux-almalinux: Connected to SSH!
==> vsphere-iso.linux-almalinux: Provisioning with Ansible...
    vsphere-iso.linux-almalinux: Setting up proxy adapter for Ansible....
==> vsphere-iso.linux-almalinux: Executing Ansible: <sensitive>-playbook -e packer_build_name="linux-almalinux" -e packer_builder_type=vsphere-iso -e packer_http_addr=172.16.38.102:8050 --ssh-extra-args '-o IdentitiesOnly=yes' --extra-vars display_skipped_hosts=false --extra-vars BUILD_USERNAME=<sensitive> --extra-vars BUILD_SECRET=***** --extra-vars ANSIBLE_USERNAME=<sensitive> --extra-vars ANSIBLE_SECRET=***** -e <sensitive>_ssh_private_key_file=/tmp/<sensitive>-key2890125948 -i /tmp/packer-provisioner-<sensitive>1495634203 /home/localadmin/packer-examples-for-vsphere/<sensitive>/main.yml
    vsphere-iso.linux-almalinux:
    vsphere-iso.linux-almalinux: PLAY [all] *********************************************************************
    vsphere-iso.linux-almalinux:
    vsphere-iso.linux-almalinux: TASK [Gathering Facts] *********************************************************
    vsphere-iso.linux-almalinux: fatal: [default]: FAILED! => {"<sensitive>_facts": {}, "changed": false, "failed_modules": {"setup": {"failed": true, "module_stderr": "/bin/sh: /usr/bin/python3: No such file or directory\nShared connection to 127.0.0.1 closed.\r\n", "module_stdout": "", "msg": "The module failed to execute correctly, you probably need to set the interpreter.\nSee stdout/stderr for the exact error", "rc": 127}}, "msg": "The following modules failed to execute: setup\n"}
    vsphere-iso.linux-almalinux:
    vsphere-iso.linux-almalinux: PLAY RECAP *********************************************************************
    vsphere-iso.linux-almalinux: default                    : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
    vsphere-iso.linux-almalinux:
==> vsphere-iso.linux-almalinux: Provisioning step had errors: Running the cleanup provisioner, if present...
==> vsphere-iso.linux-almalinux: Power off VM...
==> vsphere-iso.linux-almalinux: Destroying VM...
Build 'vsphere-iso.linux-almalinux' errored after 10 minutes 24 seconds: Error executing Ansible: Non-zero exit status: exit status 2

==> Wait completed after 10 minutes 24 seconds

==> Some builds didn't complete successfully and had errors:
--> vsphere-iso.linux-almalinux: Error executing Ansible: Non-zero exit status: exit status 2

==> Builds finished but no artifacts were created.
```

**Comment by DSI-BenThomas** [comment]
> I'll run a test on AlmaLinux in my testbed now. It's been working for quite some time now with the following unless something has changed recently in 8.5.
> 
> **Ryan Johnson** Staff II Solutions Architect | VMware, Inc.

Thanks @tenthirtyam, in my own testing updating the interpreter variable to `/usr/libexec/platform-python` resulted in the build completing successfully. Changing it back to the original `/usr/bin/python3` results in build failures again.

https://github.com/vmware-samples/packer-examples-for-vsphere/blob/e9e03fd84ddec8450ebe4ab885c0e6fbe2c5caa7/builds/linux/almalinux/8/linux-almalinux.pkr.hcl#L138-L141

**Comment by tenthirtyam** [comment]
I can confirm that I'm seeing the same. I'll issue a patch to `main`.

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.

**Comment by tenthirtyam** [comment]
Resolved in `main` with  #182.

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.
